### ISSUE TITLE
Feature request: Show hidden frontmatter

### ISSUE THREAD

**Issue by FootPrintStudio** [jparkerweb/rich-foot#13]
I like to visualize some of the "under-the-hood" frontmatter in a footer format, stuff I use for organizing and navigating my notes. Currently, I'm using a combination of admonitions and dataview to do something as seen below. Would it be possible to add this type of functionality to Rich Foot? I'm thinking along the lines of adding an extra section below the visibility settings, where you can add frontmatter properties to display with the option of adding placeholder text in case a property is empty. Ideally, this would be displayed above the backlinks and outlinks, so it doesn't get lost beneath them if there's a lot.

Read View:
![image](https://github.com/user-attachments/assets/36740719-da10-41a3-9f0a-51ad74181375)
Source View:
![image](https://github.com/user-attachments/assets/4ffe369d-ec71-429b-ad73-ffc8799326e4)

**Comment by jparkerweb** [comment]
@FootPrintStudio I'm not sure I fully understand your feature request. Are you wanting an option to display specified frontmatter fields and their values in the "Rich Foot" element in the bottom of your notes if the specified element exists???

**Comment by FootPrintStudio** [comment]
Yes, that's correct, with, if possible, the addition to provide some sort of placeholder text if the field doesn't exist or is empty.
